examples/build_sample_experiment.py

```diff
@@ -45,9 +46,11 @@ def __call__(self, spec_json):
         arg = parser.add_argument(
             "--{}-dimensions".format(aux_image_name),
             type=StarfishIndex(),
-            help="Dimensions for the {} images.  Should be a json dict, with {}, {}, and {} as the possible keys.  The "
-                 "value should be the shape along that dimension.  If a key is not present, the value is assumed to be "
-                 "0.".format(aux_image_name, Indices.ROUND, Indices.CH, Indices.Z))
+            help="Dimensions for the {} images.  Should be a json dict, with {}, {}, and {} as the "
+                 "possible keys.  The value should be the shape along that dimension.  If a key is "
+                 "not present, the value is assumed to be 0.".format(
+                aux_image_name, Indices.ROUND, Indices.CH, Indices.Z))
+        # TODO ttung: parser.add_argument doesn't return anything, so this seems odd
```

Comment:
It actually does (the action object).  I guess one could say it's not documented...

Reply:
Got it. I will remove the #TODO. Thanks.